cf-check/cf-check.c

```diff
@@ -0,0 +1,29 @@
+#include <lmdump.h>
+#include <string_lib.h>
+
+int main(int argc, char **argv)
+{
+    if (StringEndsWith(argv[0], "lmdump"))
```

Comment:
You could use `StringSafeEqual(basename(argv[0]), "lmdump"))` here. But I guess it's not a big deal.